src/sidebar.css

```diff
@@ -49,3 +51,39 @@ body {
 li.active > label {
   font-weight: bold;
 }
+
+.hidden,
+.filtered {
+  display: none;
+}
+
+#counter {
+  float: left;
```

Comment:
Please don't use float. Instead make the parent have `display: flex` and use that instead. If you are unfamiliar with flex box display I recommend [Flexbox Froggy](http://flexboxfroggy.com/) which is informative *and* fun. :)